plugins/woocommerce-blocks/assets/js/blocks/coming-soon/styles.tsx

```diff
@@ -129,7 +135,7 @@ export const generateStyles = ( color = '#bea0f2' ) => {
         height: 40px;
     }
     .woocommerce-coming-soon-banner {
-        font-size: 48px;
+        font-size: clamp(27.894px, 1.743rem + ((1vw - 3.2px) * 2.094), 48px);
```

Comment:
I set the font size to `48px` in the site editor and copied-pasted this value from the applied CSS.

The syntax for `clamp()` is:

```css
clamp(minimum, preferred, maximum)
```



- Minimum Value: 27.894px
- Preferred Value: 1.743rem + ((1vw - 3.2px) * 2.094)
- Maximum Value: 48px

Reply:
Nitpick: It sounds like you copied the values out of the editor, but perhaps we should round the numbers to ease readability 🤔

Reply:
Sounds good! Thanks for the suggestion.

Reply:
Updated in https://github.com/woocommerce/woocommerce/pull/47491/commits/f6bd20e5c3ee194694fd527948853c748a21cdeb